rd_ui/app/vendor_scripts.html

```diff
@@ -39,6 +39,9 @@
 <script src="/bower_components/canvg/StackBlur.js"></script>
 <script src="/bower_components/canvg/canvg.js"></script>
 <script src="/bower_components/leaflet/dist/leaflet.js"></script>
+<script src="/bower_components/angular-simple-logger/dist/angular-simple-logger.js"></script>
```

Comment:
This is a dependency from [angular-ui/ui-leaflet](https://github.com/angular-ui/ui-leaflet/blob/master/package.json#L76) :/